liquibase-core/src/main/java/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParser.java

```diff
@@ -138,7 +139,7 @@ public DatabaseChangeLog parse(String physicalChangeLogLocation, ChangeLogParame
                             }
                             continue;
                         } catch (NumberFormatException | NullPointerException nfe) {
-                            throw new ChangeLogParseException("Unknown ignoreLines syntax in changeset " + changeSet.toString(false));
+                            throw new ChangeLogParseException("Unknown ignoreLines syntax");
```

Comment:
What was the Sonar issue resolved here?  Did it involve the ChangeSet class?  Is that why it was removed?

Reply:
Sonar was complaing that changeSet.toString(false) can produce NPE because it is never initialized and it's true it's alway null